sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisResult.java

```diff
@@ -20,18 +20,17 @@
  *     Composed of a list of sensitivity values in pre-contingency and post-contingency states.
  * </p>
  *
- * A single sensitivity analysis should return, besides its status and some stats on the
- * analysis itself, all the sensitivity values for each factor (combination of a monitored equipment or bus and a specific
- * equipment or group of equipments). The chosen sensitivity provider offers the possibility to calculate the sensitivity
+ * Sensitivity analysis is used to assess the impact of a small modification of a network variables on the value of
+ * network functions. A combination of a variable and a function is called a sensitivity factor. A sensitivity analysis returns
+ * the sensitivity values for each factor. The sensitivity analysis API offers the possibility to calculate the sensitivities
```

Comment:
It returns the sensivity values for each factor, and the reference values for each function